index.js

```diff
@@ -3,6 +3,22 @@
 const program = require('commander');
 const fs = require('fs');
 const package = require('./package.json');
+const configName = './.pastshotsrc';
+
+function readConfig() {
+  if (fs.existsSync(configName)) {
+    try {
+      return JSON.parse(fs.readFileSync(configName, 'utf8'));
+    } catch(e) {
+      console.error('Can not parse .pasthostsrc');
+      console.error(e);
+      process.exit(1);
+    }
+  } else {
+    return JSON.parse('{}');
```

Comment:
Seems that an easier approach to this is to just return an empty object 😄 You can reduce the code nesting by testing the opposite and returning early:

```js
if (!fs.existsSync(configName)) {
  return {};
}

...
```